src/resolvers.ts

```diff
@@ -1,27 +1,50 @@
+import { User } from './user';
+import { DataSource } from 'typeorm';
+
+const AppDataSource = new DataSource({
+  type: 'postgres',
+  host: 'localhost',
+  port: 5432,
+  username: 'gbm',
+  password: '123l',
+  database: 'localserver',
+  entities: [User],
+  synchronize: true,
+});
+
 interface UserInput {
   name: string;
   email: string;
   password: string;
   birthDate: string;
 }
 
-interface User {
-  id: number;
-  name: string;
-  email: string;
-  birthDate: string;
-}
-
 export const resolvers = {
+  Query: {
+    hello: () => {
+      return 'Hello World';
+    },
+  },
   Mutation: {
     createUser: async (_parent: never, args: { data: UserInput }) => {
-      const newUser: User = {
-        id: 1,
-        name: args.data.name,
-        email: args.data.email,
-        birthDate: args.data.birthDate,
+      await AppDataSource.initialize();
+      console.log('Conectado ao db local!');
```

Comment:
No seu arquivo de setup já tem a conexão com o banco, não? E lá é um melhor lugar pra ela ficar mesmo, primeiro porque ela é relativamente demorada, e segundo que vc n precisa fazer essa conexão com o banco sempre que a mutation for chamada, vc pode fazer uma vez só, quando o sistema for iniciado. Faz sentido?